src/sessions.md

```diff
@@ -50,3 +50,7 @@ The default session column indicates which session should be preferred for new r
 
 Note that once the last phase has been reached, the X25519 key pairs for the previous session are
 no longer needed, and nodes should discard them.
+
+// TODO here would be good to have the explanation that phase 0 is at the very start of a session
+// and state that session duration are defined by the runtime blockchain api (redundant with runtime-interface.md but helps the reading).
```

Comment:
This is stated in the first sentence of this chapter: "The blockchain runtime divides time up into sessions"

Reply:
Ok, I did not really relate that phase 0 is the first of a session, but as you point this it make sense, just wasn't sure during my reading.